uber/configspec.ini

```diff
@@ -756,6 +756,10 @@ placeholder_deadline  = string(default="2016-01-18")
 # blank to keep the preregistration page open through the event's end date (eschaton).
 prereg_takedown = string(default="2016-01-18")
 
+# Before this date, attendees with single paid badges may perform self-service
+# automated refunds. Leave it blank to disable self-service refunds, period.
+refund_cutoff = string(default="")
```

Comment:
This isn't a code/design question, I'm just curious how we'd use the cutoff date. I assume we'd probably want to set this pretty close to the event epoch?

Reply:
I think the majority of events would do that, but in our case we were talking about just extending refunds until after hotels launch.